sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapExamples.cs

```diff
@@ -175,5 +176,90 @@ public void CreditDefaultSwapGetPortfolioCashFlowsExample(ModelSelection.ModelEn
             var cds = InstrumentExamples.CreateExampleCreditDefaultSwap();
             CallLusidGetPortfolioCashFlowsEndpoint(cds, model);
         }
+
+        [TestCase("2020-06-05T00:00:00.0000000+00:00")] // calculate upfront charge for cds contract
+        public void CreditDefaultSwapIsdaCdsValuationExample(string sTestNow)
+        {
+            // Purpose: Demo the valuation of a credit default swap using the IsdaCds vendor library via Lusid
+            var testScope = Guid.NewGuid().ToString();
+            var testNow = DateTimeOffset.Parse(sTestNow);
+            var cds = InstrumentExamples.CreateExampleCreditDefaultSwap() as CreditDefaultSwap;
+
+            // CREATE the configuration Recipe
+            var pricingOptions = new PricingOptions();
+            // tell Lusid that CDS instruments should be valued with the default model in IsdaCds
+            var modelRules = new List<VendorModelRule>
+                {new VendorModelRule(VendorModelRule.SupplierEnum.IsdaCds, "VendorDefault", "CreditDefaultSwap")};
+            var pricingContext = new PricingContext(options: pricingOptions, modelRules: modelRules);
+            // tell Lusid where to find data for IsdaCreditCurves; the IsdaYieldCurve is found according to a default rule
+            var mktRules = new List<MarketDataKeyRule> {new MarketDataKeyRule("Credit.IsdaCreditCurve.*.*", "Lusid", testScope, MarketDataKeyRule.QuoteTypeEnum.Rate, "mid")};
+            var mktOptions = new MarketOptions("Lusid", null, testScope);
+            var mktContext = new MarketContext(mktRules, options: mktOptions);
+            var recipeName = "IsdaCdsRecipe";
+            var recipe = new ConfigurationRecipe(
+                testScope,
+                recipeName,
+                mktContext,
+                pricingContext,
+                description: "Isda cds valuation demo"
+            );
+
+            // UPSERT the configuration recipe
+            var upsertRecipeRequest = new UpsertRecipeRequest(recipe, null);
+            var upsertRecipeResponse = _recipeApi.UpsertConfigurationRecipe(upsertRecipeRequest);
+            Assert.That(upsertRecipeResponse.Value, Is.Not.Null);
+
+            // CREATE the required market data, in this case a credit curve and a yield curve
+            string ycXml = File.ReadAllText("../../../tutorials/Ibor/ExampleMarketData/IsdaYieldCurve_USD_20200605.xml");
```

Comment:
Comment to the tune of "the IsdaYieldCurve can be sourced from the daily set published by ISDA and the credit spreads from an appropriate market data provider or internal marks"

Reply:
done